packages/stylelint-config-base/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@oacore/stylelint-config-base",
-  "version": "1.0.6",
-  "description": "Global stylelint configuration across our projects",
+  "version": "2.0.0",
```

Comment:
Version is updated automatically when you release.

Reply:
Oh, so it will update every version? I wanted to update only this package

Reply:
All packages are updated. It's easier that way